nupic/algorithms/CLAClassifier.py

```diff
@@ -213,6 +213,31 @@ def __setstate__(self, state):
     self._version = BitHistory.__VERSION__
 
 
+  def write(self, proto):
+    proto.id = self._id
+
+    statsProto = proto.init("stats", len(self._stats))
```

Comment:
It's `map<int, Real64>` in c++, which is fine if there's not a contiguous range of keys.  If there is, we should switch it to be an array in c++.  If not, we should adapt the Python implementation to use a dict or other mapping for consistency.